docs/gallery/analysis_examples/short_square_analysis.py

```diff
@@ -23,17 +25,22 @@
 sweep_info = ct.get_ephys_sweeps(specimen_id)
 
 # build a data set and find the short squares
-data_set = AibsDataSet(sweep_info=sweep_info, nwb_file=nwb_file)
-shsq_table = data_set.filtered_sweep_table(stimuli=data_set.ontology.short_square_names)
+data_set = create_data_set(sweep_info=sweep_info, nwb_file=nwb_file)
+shsq_table = data_set.filtered_sweep_table(
+    stimuli=data_set.ontology.short_square_names
+)
 shsq_sweep_set = data_set.sweep_set(shsq_table.sweep_number)
 
-# Estimate the dv cutoff and threshold fraction (we know stimulus starts at 0.27s)
-dv_cutoff, thresh_frac = ft.estimate_adjusted_detection_parameters(shsq_sweep_set.v,
-                                                                   shsq_sweep_set.t,
-                                                                   0.27, 0.271)
+# Estimate the dv cutoff and threshold fraction 
+# (we know stimulus starts at 0.27s)
+dv_cutoff, thresh_frac = estimate_adjusted_detection_parameters(
+    shsq_sweep_set.v, shsq_sweep_set.t, 0.27, 0.271
```

Comment:
These should be adjusted so that the detection parameters are actually estimated correctly. It's supposed to be right at the start of the (usually large) step current stimulus, because that rapid change in voltage can sometimes look like the start of a spike (and we don't want a spurious detection there).